llvm/CMakeLists.txt

```diff
@@ -919,14 +919,22 @@ if(NOT MSVC OR LLVM_BUILD_LLVM_DYLIB_VIS)
   set(CAN_BUILD_LLVM_DYLIB ON)
 endif()
 
-cmake_dependent_option(LLVM_LINK_LLVM_DYLIB "Link tools against the libllvm dynamic library" OFF
+# Link the tools against the libllvm DSO by default.
+set(LLVM_LINK_LLVM_DYLIB_default ON)
```

Comment:
As mentioned on the RFC thread, the deployment story on AIX does not work well with such a configuration (due to lack of relative rpath support).

Reply:
I opted AIX out of the change and updated the release notes. I expect this will not be the last platform opt-out.